source/core/Ky.ts

```diff
@@ -70,7 +70,7 @@ export class Ky {
 				const response = (await result).clone();
 
 				if (type === 'json') {
-					if (response.status === 204) {
+					if (response.status === 202 || response.status === 204) {
```

Comment:
If I'm not mistaken, in the case of a 202 response, this will work as expected if the response does not contain a body, but it will drop the body if one is present.